example/lib/widgets/buy_sheet.dart

```diff
@@ -79,7 +79,10 @@ class BuySheetState extends State<BuySheet> {
         await _onStartCardEntryFlow();
         // OR call _onStartCardEntryFlowWithBuyerVerification to start Card Entry with buyer verification (SCA)
         // NOTE this requires _squareLocationSet to be set
-        // await _onStartCardEntryFlowWithBuyerVerification();
+         //await _onStartCardEntryFlowWithBuyerVerification();
```

Comment:
nit: fix the indent and space after `//`

Reply:
Sure.